aicsimageio/readers/ome_tiff_reader.py

```diff
@@ -197,10 +200,10 @@ def _get_dims_and_coords_from_ome(
         # Time
         # If global linear timescale we can np.linspace with metadata
         if scene_meta.pixels.time_increment is not None:
-            coords[DimensionNames.Time] = np.linspace(
+            coords[DimensionNames.Time] = np.arange(
                 0,
-                scene_meta.pixels.time_increment_quantity,
-                scene_meta.pixels.size_t,
+                scene_meta.pixels.size_t * scene_meta.pixels.time_increment,
+                scene_meta.pixels.time_increment,
```

Comment:
Lol, @toloudis looks like I had already fixed that time coordinate plane bug on this branch and forgot about it...